packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.ts

```diff
@@ -7,6 +7,7 @@ import { Component, HostBinding, Input, OnInit } from '@angular/core';
 export class ButtonComponent implements OnInit {
   @Input() type: 'submit' | 'button' = 'button';
   @Input() fullWidth: boolean | string = false;
+  @Input() isDisabled: boolean | string = false;
```

Comment:
Also doesn't have to be on this PR, but we should have types be just `boolean` instead of `boolean | string`.